cli/src/main.rs

```diff
@@ -252,6 +252,15 @@ fn run() -> Result<(), CliError> {
                                 .long("member")
                                 .help("Filter the circuits by a node ID in the member list")
                                 .takes_value(true),
+                        )
+                        .arg(
+                            Arg::with_name("format")
+                                .short("f")
+                                .long("format")
+                                .help("Format of the circuit")
+                                .possible_values(&["default", "csv"])
+                                .default_value("default")
```

Comment:
Need a descriptive name that indicates what the default format is, such as `table` or `text`.  ( See the AWS for the differences between text and table format.)

Reply:
P.S.  Here's the link for the AWS doc: https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-output.html

Reply:
We don't want a more descriptive name. Our output approach for the default could change over time.